packages/react-next/etc/react-next.api.md

```diff
@@ -2555,6 +2486,7 @@ export * from "@fluentui/react-button/lib/Button";
 export * from "@fluentui/react-button/lib/MenuButton";
 export * from "@fluentui/react-button/lib/SplitButton";
 export * from "@fluentui/react-button/lib/ToggleButton";
+export * from "@fluentui/react-slider/lib/Slider";

```

Comment:
I'm wondering why for subpackages we don't export * from the package? Is there any reason why we would be using the top level physical files?

I think it could make downstream testing more difficult when testing with commonjs. In commonjs, this import won't work without some form of pre-bundling or redirect.

Reply:
I know for AMD partners they might need to do extra work to resolve the packages, possibly. But that's the nature of AMD with the lacking in node-compatible resolution. I think out of the box we should prioritize esm and node (commonjs) or environments.

Reply:
this is not an issue since we have the correct build/output in `lib-commonjs` today

Reply:
The only reason we have the correct lib-commonjs output is because of this postprocess script https://github.com/microsoft/fluentui/blob/master/scripts/tasks/postprocess-commonjs.js

Reply:
And we have a similar one for AMD build postprocessing